src/Jackett.Common/Indexers/BakaBT.cs

```diff
@@ -23,12 +25,14 @@ public class BakaBT : BaseWebIndexer
         private string SearchUrl => SiteLink + "browse.php?only=0&hentai=1&incomplete=1&lossless=1&hd=1&multiaudio=1&bonus=1&reorder=1&q=";
         private string LoginUrl => SiteLink + "login.php";
         private readonly string LogoutStr = "<a href=\"logout.php\">Logout</a>";
+        private bool AddRomajiTitle => configData.AddRomajiTitle.Value;
+        private bool AppendSeason => configData.AppendSeason.Value;
```

Comment:
This will cause a null pointer for users with previous configuration.We have to try to be backwards compatible.
1. Remove the tracker from Jackett.
2. Go to a previous version of Jackett and configure BakaBT
3. Start Jackett with this code and click in "search" or "test" without changing the configuration.

You will see a null pointer exception and a regular user will report the issue because it's not clear that he has to update the configuration.
Checking `configData.AppendSeason != null && configData.AppendSeason.Value` will be enough. You are using each var only once so maybe you don't need these declarations.

Reply:
Thank you for the review.

I have actually tested the steps you suggested, but I am not receiving any errors/exceptions.

I think both `configData.AddRomajiTitle` and `configData.AppendSeason` are initialized correctly in `ConfigurationDataBakaBT`'s constructor and they are never null unless `ConfigurationDataBakaBT` is not initialized first.

I have previously done the same implementation for Animebytes, and I don't think any users are facing the aforementioned issue.

Reply:
I had to do some changes in the Gazelle tracker to handle that issue => https://github.com/Jackett/Jackett/pull/9666/commits/39c24bb494b95c3f43eae217c48664bdf4ff04f3
I can't test this tracker but if you did and it's ok I'm going to merge. Thank you!